light-client/src/node.rs

```diff
@@ -31,6 +31,19 @@ impl Node {
         })
     }
 
+    pub fn with_portal(config: Arc<Config>, portal_rpc: R) -> Result<Self, NodeError> {
+        let checkpoint_hash = &config.checkpoint.as_ref().unwrap();
```

Comment:
hmmmm... maybe in a future pr, but I'd like to see the clippy lints we use in our other workspaces added to this workspace
```rust
#![warn(clippy::unwrap_used)]
#![warn(clippy::uninlined_format_args)]
```

Reply:
Will create an issue to enable `#![warn(clippy::unwrap_used)]` and fix the unwraps.